Runtime/RenderPipeline/PSXRenderPipelineAsset.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace HauntedPSX.RenderPipelines.PSX.Runtime
 {
-    public partial class PSXRenderPipelineAsset : RenderPipelineAsset
+    public partial class PSXRenderPipelineAsset : RenderPipelineAsset<PSXRenderPipeline>
```

Comment:
Did this manually. Seems to be required (warning logs in Unity specifically say to do this)